mfr/extensions/utils.py

```diff
@@ -0,0 +1,22 @@
+from urllib.parse import parse_qs, urlencode
+
+from mfr.extensions import settings
+
+
+def get_local_export_url(urlparsed_url):
+    """
+    Replace the docker domain with localhost (for use in local development).
+
+    :param parsed_url: the default export URL which is parsed by `urllib.parse.urlparse`
+    :return: the tweaked export URL for local development
+    """
+
+    query_dict = parse_qs(urlparsed_url.query, keep_blank_values=True)
+    # TODO: why do we pop `mode`?
```

Comment:
???  Did this come from somewhere else?

Reply:
That was a question from @cslzchen that I couldn't remember the answer to.